lib/common/entropy_common.c

```diff
@@ -85,6 +85,8 @@ size_t FSE_readNCount (short* normalizedCounter, unsigned* maxSVPtr, unsigned* t
     }   }
     assert(hbSize >= 4);
 
+    /* init */
+    memset(normalizedCounter, 0, (*maxSVPtr+1) * sizeof(normalizedCounter[0]));   /* all symbols not present in NCount have a frequency of 0 */
```

Comment:
Is this the same speed/faster than just memsetting the unset symbols at the end?

Reply:
It's the same (aka : invisible, same speed as if no `memset()` was present).

I've favored this version because, it's the same speed, but this one is simpler,
and does not need to wait for a dependency to proceed.